### PR TITLE
write out rds file for all sites

### PR DESCRIPTION
**Pull request by mhines-usgs**
@jread-usgs here's the wqp site info to nhd lookup, written out to rds. 

I left out the nhd_ prefix because this function is kind of specific to those data, but if you want that prefix there so it's consistent with other files, I can do that. Let me know.

### CODE REVIEW

File: scripts/getSiteData.R [hunk 1]
```diff
@@ -45,7 +45,8 @@ for (j in 1:length(states)) {
     }
   })
 } 
-#remove NA values
-matches <- subset(lookup,!is.na(id))
-write.csv(matches, file=paste0(getwd(),"/data/wqp_nhd",".csv"),row.names=FALSE)
+
+
+saveRDS(lookup, file=paste0(getwd(),"/data/wqp_nhd/wqp_nhdLookup.rds"))
```
Comment (jordansread) [inline scripts/getSiteData.R]: `file="data/wqp_nhd/wqp_nhdLookup.rds"` instead?

**Comment by jordansread** [comment]
@mhines-usgs what changed in `data/NHD_summ/nhd_centroids.csv`?

**Comment by jordansread** [comment]
I don't see any code changes that reference that file

File: scripts/getSiteData.R (hunk 1 above)
Comment (mhines-usgs) [inline scripts/getSiteData.R]: k

**Comment by jordansread** [comment]
should this eventually end up in the wqp make or the nhd one?

**Comment by lawinslow** [comment]
I'd say WQP. I think of the NHD one as getting the shapefiles and prepping them for linking to other data.

**Comment by mhines-usgs** [comment]
There aren't any obvious differences in the centroid file, I don't know why it came up differently, other than the first time you (@jread-usgs) committed it, and this time I did. Perhaps there is a windows difference?

**Comment by lawinslow** [comment]
Maybe just a different order?

**Pull request merged**